filters/general/filter_24_1Hosts_Lite/metadata.json

```diff
@@ -1,6 +1,7 @@
 {
-  "filterId": "1hosts_lite",
```

Comment:
Wouldn't it be better to have the same structure as in FiltersRegistry?
https://github.com/AdguardTeam/FiltersRegistry/blob/master/filters/filter_11_Mobile/metadata.json

I.e. `id -> filterId`

Reply:
Will do.